custom-schemes.md

```diff
@@ -8,19 +8,25 @@ Existing specifications like [OpenID4VP](https://openid.net/specs/openid-4-verif
    - This seems to increase the opportunity for attacks such as replay attacks, man-in-the-middle attacks and phishing. It also seems to limit the opportunity for abuse detection and prevention.
    - A browser API can securely pass the requesting origin on to the wallet so that the wallet can know with confidence what origin the user was on when the request was initiated.
    
-## How are wallet apps supposed to pass info back to the invoking page?
-
-  - Does the page give the wallet a URL to navigate back to as part of the custom scheme URL it uses to launch the wallet?
-
-  - If so, that's a problem because it won't go back to the same tab. Apps don't have the ability to return information to the same page that invoked them after a custom scheme invocation; they can just open a new page in a new tab.
-
 ## Can wallets limit requests to secure contexts?
 
    - If a user uses a website via http (not https) and it invokes a custom scheme, it's possible that an attacker with access to the network could eavesdrop on or even modify the request (such as by replacing the verifier's public key with their own).
    - A browser API could (and almost certainly should) take the position that it will function only within the context of a secure context (https).
 
 ## User experience concerns
 
+### What is the user experience of the wallet operation?
```

Comment:
Is this redundant with the "Friction and app switching" bullet point?

Reply:
Oh whoops, good point. Thanks, I merged them.